README.md

```diff
@@ -19,11 +19,42 @@ different technologies.
 Find more information about the client configuration in the README file of the repo:
 [https://github.com/superdesk/superdesk-client-core](https://github.com/superdesk/superdesk-client-core "") 
 
-### Installation
+## Installation
+
+### Client
+
+1. Clone the repository
+2. Navigate to the folder where you've cloned this repository (if it's the main repo, go inside the `client` folder).
+2. Run `npm install && bower install` to install dependencies.
+3. Run `grunt server` to run the web server.
```

Comment:
after this you could add a step to open browser and navigate to `localhost:9000`